### ISSUE TITLE
Issue on servers

### ISSUE THREAD

**Issue by Soul-less01** [mekanism/Mekanism#7219]
```
[29May2021 15:16:35.942] [Server thread/ERROR] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: Exception caught during firing event: null
	Index: 10
	Listeners:
		0: NORMAL
		1: ASM: class com.feed_the_beast.mods.ftbessentials.FTBEEventHandler playerLoggedIn(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		2: ASM: class vazkii.patchouli.common.handler.PlayerJoinHandler playerLogin(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		3: ASM: class net.geforcemods.securitycraft.SCEventHandler onPlayerLoggedIn(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		4: ASM: class sonar.fluxnetworks.register.CommonEventHandler onPlayerJoined(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		5: net.minecraftforge.eventbus.EventBus$$Lambda$2711/880234640@20f1ae1
		6: ASM: class corgitaco.enchancedcelestials.EnhancedCelestials$EnhancedCelestialsEvents onPlayerJoined(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		7: ASM: net.minecraftforge.common.ForgeInternalHandler@18cb0b5d playerLogin(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		8: net.minecraftforge.eventbus.EventBus$$Lambda$2711/880234640@6825d0e2
		9: ASM: class net.mehvahdjukaar.supplementaries.world.data.GlobeData onPlayerLoggedIn(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		10: ASM: mekanism.common.CommonPlayerTracker@33c1c01 onPlayerLoginEvent(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
java.lang.NullPointerException
	at net.minecraft.network.PacketBuffer.func_179252_a(PacketBuffer.java:224)
	at mekanism.common.network.PacketSecurityUpdate.encode(PacketSecurityUpdate.java:77)
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.lambda$tryEncode$4(IndexedMessageCodec.java:134)
	at java.util.Optional.ifPresent(Optional.java:159)
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.tryEncode(IndexedMessageCodec.java:132)
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.build(IndexedMessageCodec.java:147)
	at net.minecraftforge.fml.network.simple.SimpleChannel.encodeMessage(SimpleChannel.java:85)
	at net.minecraftforge.fml.network.simple.SimpleChannel.toBuffer(SimpleChannel.java:98)
	at net.minecraftforge.fml.network.simple.SimpleChannel.toVanillaPacket(SimpleChannel.java:129)
	at net.minecraftforge.fml.network.simple.SimpleChannel.sendTo(SimpleChannel.java:109)
	at mekanism.common.network.BasePacketHandler.sendTo(BasePacketHandler.java:93)
	at mekanism.common.CommonPlayerTracker.onPlayerLoginEvent(CommonPlayerTracker.java:41)
	at net.minecraftforge.eventbus.ASMEventHandler_286_CommonPlayerTracker_onPlayerLoginEvent_PlayerLoggedInEvent.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
	at net.minecraftforge.fml.hooks.BasicEventHooks.firePlayerLoggedIn(BasicEventHooks.java:44)
	at net.minecraft.server.management.PlayerList.func_72355_a(PlayerList.java:231)
	at net.minecraft.network.login.ServerLoginNetHandler.func_147326_c(ServerLoginNetHandler.java:118)
	at net.minecraft.network.login.ServerLoginNetHandler.func_73660_a(ServerLoginNetHandler.java:65)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:222)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:134)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:865)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787)
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642)
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232)
	at java.lang.Thread.run(Thread.java:748)

[29May2021 15:16:35.944] [Server thread/WARN] [net.minecraft.network.NetworkSystem/]: Failed to handle packet for /10.0.0.88:54372
java.lang.NullPointerException: null
	at net.minecraft.network.PacketBuffer.func_179252_a(PacketBuffer.java:224) ~[?:?]
	at mekanism.common.network.PacketSecurityUpdate.encode(PacketSecurityUpdate.java:77) ~[mekanism:10.0.21]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.lambda$tryEncode$4(IndexedMessageCodec.java:134) ~[forge:?]
	at java.util.Optional.ifPresent(Optional.java:159) ~[?:1.8.0_292]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.tryEncode(IndexedMessageCodec.java:132) ~[forge:?]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.build(IndexedMessageCodec.java:147) ~[forge:?]
	at net.minecraftforge.fml.network.simple.SimpleChannel.encodeMessage(SimpleChannel.java:85) ~[forge:?]
	at net.minecraftforge.fml.network.simple.SimpleChannel.toBuffer(SimpleChannel.java:98) ~[forge:?]
	at net.minecraftforge.fml.network.simple.SimpleChannel.toVanillaPacket(SimpleChannel.java:129) ~[forge:?]
	at net.minecraftforge.fml.network.simple.SimpleChannel.sendTo(SimpleChannel.java:109) ~[forge:?]
	at mekanism.common.network.BasePacketHandler.sendTo(BasePacketHandler.java:93) ~[mekanism:10.0.21]
	at mekanism.common.CommonPlayerTracker.onPlayerLoginEvent(CommonPlayerTracker.java:41) ~[mekanism:10.0.21]
	at net.minecraftforge.eventbus.ASMEventHandler_286_CommonPlayerTracker_onPlayerLoginEvent_PlayerLoggedInEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.fml.hooks.BasicEventHooks.firePlayerLoggedIn(BasicEventHooks.java:44) ~[forge:?]
	at net.minecraft.server.management.PlayerList.func_72355_a(PlayerList.java:231) ~[?:?]
	at net.minecraft.network.login.ServerLoginNetHandler.func_147326_c(ServerLoginNetHandler.java:118) ~[?:?]
	at net.minecraft.network.login.ServerLoginNetHandler.func_73660_a(ServerLoginNetHandler.java:65) ~[?:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:222) ~[?:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:134) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:865) ~[?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_292]
```

The last time the server worked was `[29May2021 01:16:17.708]`, the error appeared ten hours later at `[29May2021 11:39:52.736]`. I woke up to find my members couldnt log in, so i tried and the error above is what was produced. I'm sure the error is mekanism related due to the `Failed to handle packet for <local ip>` block showing `mekanism.common.network.BasePacketHandler.sendTo`

**Comment by Soul-less01** [comment]
I have the latest version of Mekanism installed, as well.

**Comment by Soul-less01** [comment]
Sorry about no format, I posted it in a hurry and didn't even know there was a format.

> Issue description:
Randomly my server stopped allowing players to join, producing an error pointing to Mekanism.

> Steps to reproduce:
Honestly no clue, it randomly happened. Check the log from the first post.

> Version (make sure you are on the latest version before reporting):
Forge: 36.1.2(3?)
Mekanism: 10.0.21

> Video:
None, all it would show is a player connecting to the server to only be disconnected with a "Internal Server Error"

> Misc:
The error was fixed by reverting the world to an older backup.
There are no logs from the last point it was working to when the error occurred.
